finality-aleph/src/network/clique/protocols/v1/mod.rs

```diff
@@ -28,6 +27,19 @@ enum Message<D: Data> {
     Heartbeat,
 }
 
+async fn check_authorization<SK: SecretKey>(
```

Comment:
Huh, what? This is part of @fixxxedpoint 's task, right? Why do I see it in this diff?

Reply:
moved from `v0/mod.rs`

Reply:
Ah...